docs/install_guides/installing-openvino-yum.md

```diff
@@ -61,15 +73,15 @@ Results:
 intel-openvino-2021 Intel(R) Distribution of OpenVINO 2021
 ```
   
-### To list the available OpenVINO packages
+### To list available OpenVINO packages
 Use the following command:
 ```sh
 yum list intel-openvino*
 ```
 
 ---
   
-## Install the runtime packages Using the YUM Package Manager
+## Install Runtime Packages Using the YUM Package Manager
 
 Intel® OpenVINO will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<BUILD_NUM>`
 <br>
```

Comment:
Symlink name shall be corrected to a format /opt/intel/openvino_<VERSION>:

A symlink will be created: `/opt/intel/openvino_<VERSION>`

Please update corresponding line for APT page also.

Reply:
@slyubimt Fixed, please check.